### PR TITLE
Emphasize use of state variable for custom devices

### LINKED ISSUE #20558: Variable used for ISY994 custom device must be a state variable
### Feedback

The ISY994 Integration provides support for custom devices using programs and variables. There are two types of variables available in the ISY994 and the documentation does not make clear that the variable used for this purpose **must** be a state variable rather than an integer variable.

There is also:
- use of the phrase "Name of entity to send the command" where "Name of target entity for the command" would be more appropriate (the entity is not sending the command, it is receiving it).
- a few other minor grammatical errors

I will submit a pull request with suggested fixes.

### URL

https://www.home-assistant.io/integrations/isy994/

### Version

_No response_

### Additional information

This is related to issue #60804, submitted against the Home Assistant core (but turned out to be just a documentation issue).

### PR DESCRIPTION
**Pull request by tktf50**
## Proposed change

This documentation update is primarily to emphasize the need to use a state variable (not an integer variable) when configuring a custom component. Also fixed a few grammatical errors.

## Type of change

- [x] Spelling, grammar or other readability improvements (`current` branch).
- [x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: #20558

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Comment by homeassistant** [comment]
Hi @TKTF50, 

It seems you haven't yet signed a CLA. Please do so [here](https://home-assistant.io/developers/cla_sign_start/?pr=home-assistant&#x2F;home-assistant.io#20559).

Once you do that we will be able to review and accept this pull request.

Thanks!

File: source/_integrations/isy994.markdown [hunk 1]
```diff
@@ -330,13 +330,19 @@ A device is created by creating a directory, with the name for the device, under
...
 
 <p class='img'>
   <img src='/images/isy994/isy994_CoverExample.png' />
 </p>
 
-The *status* program in this directory is what indicates the state of the device:
+The *status* program requires that you create a state variable with the name of your choice. Note that you must use a state variable, not an integer variable. This variable will store the actual status of the new device and will be updated by the *action* program.
+
+<p class='img'>
+  <img src='/images/isy994/isy994_SwitchVariableExample.png' />
```
Comment (frenck) [inline source/_integrations/isy994.markdown]: - Please lower care the file name and use snake case filename styling.
- Please use Markdown to embed the image.

**Review by bdraco — DISMISSED** [review]
Please see comment above. Thanks 👍 

https://github.com/home-assistant/home-assistant.io/pull/20559#discussion_r762152563

**Review by frenck — APPROVED** [review]
Thanks, @TKTF50 👍

**Pull request merged**